Jenkinsfile-scheduled-node-build

```diff
@@ -0,0 +1,41 @@
+library("tdr-jenkinslib")
+
+pipeline {
+  agent {
+    label "built-in"
+  }
+  stages {
+    stage("Rebuild nodes") {
+      steps {
+        script {
+          def nodeNames = ["aws", "npm", "plugin-updates", "postgres", "terraform", "transfer-frontend", "terraform-latest"]
+          nodeNames.each() { String jobName ->
+            build(
+                job: "TDR Build Jenkins Node Images",
```

Comment:
Indentation should be 2 spaces.

Reply:
I copied this from somewhere else, I blame Intellij. I've fixed it.